examples/annotation_import/image.ipynb

```diff
@@ -104,7 +104,7 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        global_keys=global_key, # DataRow objects, a list of global keys or data row ids are supported
</pre><p><code>global_keys=[global_key]</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1006/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>